etc/kconfigs.yaml

```diff
@@ -48,6 +48,7 @@ DAMON_LRU_SORT: v6.0-rc1
 DAMON_PADDR: v5.16-rc1
 DAMON_RECLAIM: v5.16-rc1
 DMABUF_SELFTESTS: v5.4-rc1
+DMAPOOL_TEST: v6.4-rcl
```

Comment:
the commit is for "include/kunit: add DRM_VC4_KUNIT_TEST", does this DMAPOOL_TEST need a separated commit?